docs/PremiumApps.md

```diff
@@ -0,0 +1,488 @@
+# Premium App Installation Guide
+
+The Splunk Operator currently provides support for automating the installation of Enterprise Security with support for other premium apps coming in the future. This page documents the prerequisites, installation steps, and limitations of deploying premium apps using the Splunk Operator.
+
+## Enterprise Security
+
+### Prerequisites
+
+Installing Enterprise Security in a Kubernetes cluster with the Splunk Operator requires the following:
+
+* Ability to utilize the Splunk Operator [app framework](https://splunk.github.io/splunk-operator/AppFramework.html) method of installation.
+* Access to the [Splunk Enterprise Security](https://splunkbase.splunk.com/app/263/) app package.
+* Splunk Enterprise Security version 6.4.1 or 6.6.0 as Splunk Operator requires Splunk Enterprise 8.2.2 or later. For more information regarding Splunk Enterprise and Enterprise Security compatibility, see the [version compatibility matrix](https://docs.splunk.com/Documentation/VersionCompatibility/current/Matrix/CompatMatrix).
+* If installing to an Indexer Cluster, access to the corresponding Splunk_TA_ForIndexers app from the Enterprise Security package. This app can be extracted using the procedure available at [Splunk ES app installation on Indexer Cluster](https://docs.splunk.com/Documentation/ES/7.0.2/Install/InstallTechnologyAdd-ons). After the Splunk_TA_ForIndexers is available, place it in a S3 bucket and use "cluster" scope in the YAML file to push the package via the cluster manager to the indexers.  
+* Pod resource specs that meet the [Enterprise Security hardware requirements](https://docs.splunk.com/Documentation/ES/latest/Install/DeploymentPlanning#Hardware_requirements).
+* In the following sections, aws s3 remote bucket is used for placing the splunk apps, but as given in the [app framework doc](https://splunk.github.io/splunk-operator/AppFramework.html), you can use Azure blob remote buckets also.
+
+### Supported Deployment Types
+
+Currently there are only a subset of architectures that support automated deployment of Enterprise Security through the Splunk Operator.
+
+Supported Architectures Include:
+* Standalone Splunk Instances 
+* Standalone Search Head(s) which search any number of Indexer Clusters.
```

Comment:
Should we include SHC without indexer clusters? Or should we generalize to:

1. Standalone
2. SHC
3. IDXC (with standalone SH or SHC)

Reply:
I did some updates inline with SVAs that support ES.